### ISSUE TITLE
pdf_info failing with embedded NUL

### ISSUE THREAD

**Issue by 1beb** [ropensci/pdftools#46]
image: rstudio/rocker
packages: devtools install from github
crossposted in case not a bug: https://stackoverflow.com/questions/53293124/pdftools-embeded-nul-in-string
 
I'm trying to download a file and read it's info automatically, from the following link: 

http://www.leyes.congreso.gob.pe/Documentos/2016_2021/Proyectos_de_Ley_y_de_Resoluciones_Legislativas/PL0361420181108.pdf

The problem is that when I try to read the information on the pdf, I get an error. It seems to happen on and off, I can't see a good reason why. The error appears to be Linux only. 

```
    library(pdftools)
    link = "http://www.leyes.congreso.gob.pe/Documentos/2016_2021/Proyectos_de_Ley_y_de_Resoluciones_Legislativas/PL0361420181108.pdf"
    download.file(link, "somefile.pdf")
    pdf_info("somefile.pdf")
    Error in poppler_pdf_info(loadfile(pdf), opw, upw) : 
      Embedded NUL in string.
```

What else I've tried: 

- Tried downloading using mode = "wb"
- Tried downloading with httr using the write_disk method 
- Tried downloading manually on windows and it works! :( 
- Tried downloading using curl:download() and it still failed.

My suspicion was/is that it has to do with the way I'm downloading the file. But, I don't know what alternatives I should be trying.

**Comment by jeroen** [comment]
We fixed a bunch of encoding bugs in libpoppler so I'm hopeful this will be solved in the next version of pdftools.

**Comment by 1beb** [comment]
Is this something I can test by installing directly from GitHub?

**Comment by jeroen** [comment]
Not yet maybe tomorrow. Which OS are you on?

**Comment by 1beb** [comment]
The process that is most effected by this is on Ubuntu. It doesn't happen
on Windows. (I do my dev local, then push big work to cloud, was using a
pretty typical amazon ami with rstudio on it)

**Comment by jeroen** [comment]
Can you try the dev version? 

```r
remotes::install_github("ropensci/pdftools")
```

Also please report your `pdftools::poppler_config()`.

**Comment by jeroen** [comment]
This should be fixed in pdftools 2.0.
